osu.Game/Localisation/OnlineSettingsStrings.cs

```diff
@@ -94,6 +94,11 @@ public static class OnlineSettingsStrings
         /// </summary>
         public static LocalisableString HideCountryFlags => new TranslatableString(getKey(@"hide_country_flags"), @"Hide country flags");
 
+        /// <summary>
+        /// "Broadcast game state"
```

Comment:
This means nothing to me, nor any user. It should 100% mention "websocket" in the string – something like "Enable websocket server" – and have a link to a wiki page which has documentation on how to interact with it.

I'd see that documentation page existing is part of this PR.